scripts/relup-build/download-base-packages.sh

```diff
@@ -14,7 +14,7 @@ export PROFILE
 
 case $PROFILE in
     "emqx-enterprise")
-        DIR='enterprise'
+        DIR='emqx-ee'
```

Comment:
the default profile is `emqx-enterprise`.
Personally, I suggest changing all `emqx-enteriprise` to `emqx-ee`. make the `$PROFILE=emqx-ee`